ccf_transaction_processor/generate_signing_keys/gen_keys.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage ./gen_keys.sh <ip-address-of-CCF>
+
+# Copy KEYs for test
```

Comment:
This seems a generic script, yet some operations are for "tests".
If additional work is required to separate common operations from tests, it's better to add a detailed TODO.

Reply:
thank you, some of the comments in this script are wrong (blame my copy paste). I am fixing them. this is not a test script. there is no plan to add more tests, other than testing ccf via using with along with pdo (via run tests). Thanks.